src/components/ansatte/Ansatte.tsx

```diff
@@ -1,20 +1,21 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './Ansatte.less';
-import {useAppStore} from '../../data/store/AppStore';
-import {AnsattRad} from "./AnsattRad";
-import {Validering} from "../validering/Validering";
-import {ByggValideringsFeil} from "./ByggValideringsFeil";
+import { useAppStore } from '../../data/store/AppStore';
+import { AnsattRad } from "./AnsattRad";
+import { Validering } from "../validering/Validering";
+import { ByggValideringsFeil } from "./ByggValideringsFeil";
 import Innsending from "./Innsending";
-import {LeggTilKnapp} from "./LeggTilKnapp";
-import {BekreftKnapp} from "./BekreftKnapp";
-import {HjelpetekstRefusjon} from "./HjelpetekstRefusjon";
-import {HjelpetekstDager} from "./HjelpetekstDager";
-import {HjelpetekstPeriode} from "./HjelpetekstPeriode";
-import {Eklæring} from "./Erklæring";
-import {ValideringOppsummering} from "./ValideringOppsummering";
+import { LeggTilKnapp } from "./LeggTilKnapp";
+import { BekreftKnapp } from "./BekreftKnapp";
+import { HjelpetekstRefusjon } from "./HjelpetekstRefusjon";
+import { HjelpetekstDager } from "./HjelpetekstDager";
+import { HjelpetekstPeriode } from "./HjelpetekstPeriode";
+import { Eklaring } from "./Erklaring";
```

Comment:
Typo her, den burde vel hete Erklaring, eller enda bedre; Erklaering 🎉

Reply:
Erklaring var meningen. Tenkte på Erklaering og men gikk for den annen.

Reply:
Hvis man skal være 100% korrekt er vel æ -> ae riktigst, men enig i at "erklaring" er litt enklere å lese enn "erklaering" :)